java/CMakeLists.txt

```diff
@@ -54,4 +69,72 @@ if(JNI_FOUND)
       LIBRARY DESTINATION ${JAVA_INSTALL_PATH}
     )
   endif()
-endif()
+
+  if(NOT WIN32)
+    # Bare Library targed to use for spark
+    find_package(Java)
```

Comment:
Is the Java package used?

Reply:
it's used by the JNI generating

Reply:
I commented it out and added some description to explain how to use it during development